examples/inspection/plot_partial_dependence.py

```diff
@@ -191,7 +191,7 @@
 # Analysis of the plots
 # .....................
 #
-# We can clearly see on the PDPs (thick blue line) that the median house price
+# We can clearly see on the PDPs (thick orange line) that the median house price
```

Comment:
It looks like the line is now a dashed orange line:

```suggestion
# We can clearly see on the PDPs (dashed orange line) that the median house price
```

Reply:
yes, thanks for correcting :)